<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutSnapIn</class>
 <widget class="QWidget" name="CheckoutSnapIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="checkoutOptionsPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="tTitleLabel" name="titleLabel">
         <property name="text">
          <string>Checkout Revision</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="checkoutOptionsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1" alignment="Qt::AlignTop">
           <widget class="QRadioButton" name="checkoutCommitButton">
            <property name="text">
             <string>Commit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Checkout:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="checkoutBranchButton">
            <property name="text">
             <string>Branch</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="branchBox"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="tSubtitleLabel" name="label_2">
            <property name="text">
             <string>Checkout Options</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="CommitBox" name="commitBox" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="checkoutButton">
         <property name="text">
          <string>Checkout</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>270</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tSubtitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">tsubtitlelabel.h</header>
  </customwidget>
  <customwidget>
   <class>CommitBox</class>
   <extends>QWidget</extends>
   <header>widgets/commitbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
